### PR TITLE
Added support for pgaudit extension.

### PR DESCRIPTION
**Pull request by ermajn**
I've tested build and debian packaging takes care of versioning (diferent version of extension for each postgresql version).

### CODE REVIEW

File: postgres-appliance/Dockerfile [hunk 1]
```diff
@@ -151,7 +151,8 @@ RUN export DEBIAN_FRONTEND=noninteractive \
                         postgresql-${version}-postgis-${POSTGIS_VERSION%.0} \
                         postgresql-${version}-postgis-${POSTGIS_VERSION%.0}-scripts \
                         postgresql-${version}-repack \
-                        postgresql-${version}-wal2json" \
+                        postgresql-${version}-wal2json \
+                        postgresql-${version}-pgaudit" \
```
Comment (CyberDem0n) [inline postgres-appliance/Dockerfile:155]: Please move it after hypopg: https://github.com/zalando/spilo/pull/448/files#diff-82f985fcf1262fb2d3d1843b9e74d9c3R141, so extensions are ordered alphabetically :)

Comment (ermajn) [reply postgres-appliance/Dockerfile]: ok, done. :)

**Comment by FxKu** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**